src/Unpacker2D.cc

```diff
@@ -449,7 +449,7 @@ void Unpacker2D::DistributeEventsSingleStep()
               else
               {
                 // regular ending
-                file->seekg(file->tellg() - 4);
+                file->seekg(static_cast<unsigned int>(file->tellg()) - 4);
```

Comment:
I think it is fine, the alternative approach, could be I guess:
```suggestion
                file->seekg(file->tellg(), - 4);
```

Reply:
I think there is no seekg version that accepts integer offset:
https://www.cplusplus.com/reference/istream/istream/seekg/

With the above proposition, I am getting a compilation error:
``` 
error: invalid conversion from ‘int’ to ‘std::ios_base::seekdir’
```

Reply:
Sorry, my bad. Your solution is fine